src/db/utils/get-thread-by-id/get-thread-by-id.ts

```diff
@@ -0,0 +1,9 @@
+import { ThreadModel } from "../../../models/thread/thread.model";
+/**
+ * Finds user by ID and only returns relevant properties
+ * @param userId ObjectId
+ */
+export async function getThreadById({threadId}: {threadId: string}) {
+  const result = await ThreadModel.findById(threadId);
+  return result;
```

Comment:
You could just return await here, no? 
```suggestion
  return await ThreadModel.findById(threadId);
```